src/__stories__/snackbar-story.tsx

```diff
@@ -12,13 +12,20 @@ type Args = {
     message: string;
```

Comment:
just to keep my TOC calm, can you change the order between buttonText and message in storybook control table?
![image](https://github.com/Telefonica/mistica-web/assets/6722153/1bd2a762-48c5-461c-bc53-5727a2080cec)

Order:
- type
- message
- buttonText
- withDismiss
- duration

Extra ball, can we remove "5000" & "10000" from storybook controls? This configuration is something we should deprecate in the next major

Reply:
done